package exceptionhandling;

 class Test4 extends RuntimeException {

	public static void main(String[] args)throws Test4 {
		System.out.println("10/0");

	}

}
